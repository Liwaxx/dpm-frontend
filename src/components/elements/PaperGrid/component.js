import React from 'react';
import PropTypes from 'prop-types';
import Text from '../Text';

export default class Component extends React.Component {
  render() {
    const { classes, children } = this.props;

    return (
      <div className={classes.container}>
        <div className={classes.pembuka}>
          {children}
        </div>

        <div className={classes.add}>
          <Text className={classes.add_text}><br/>ADD TASK</Text>
        </div>
      </div>
    );
  }
}

Component.propTypes = {
  classes: PropTypes.object.isRequired,
  children: PropTypes.object
};

