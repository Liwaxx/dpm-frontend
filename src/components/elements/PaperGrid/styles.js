const styles = ({
  root: {
    flexGrow: 1,
  },
  container: {
    width: '100fr',
  },
  pembuka: {
    backgroundColor: '#EBEFF5',
    width: '100%',
    height: '100%',
    padding: [0, 10],
  },
  add: {
    padding: 10,
    margin: 0,
    textAlign: 'center',
    marginBottom: 40,
    backgroundColor: '#EBEFF5',
    color: '#AFDAF2',
    width: '100%',
  },
  add_text: {
    color: '#71C4EE',
    textAlign: 'center',
  },
});

export default styles;
