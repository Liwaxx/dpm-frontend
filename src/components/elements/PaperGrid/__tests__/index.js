import React from 'react';
import renderer from 'react-test-renderer';
import PaperGrid from '../index';

jest.mock('../../Text');

describe('PaperGrid', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<PaperGrid />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});