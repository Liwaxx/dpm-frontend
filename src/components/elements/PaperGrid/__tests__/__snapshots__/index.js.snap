// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaperGrid renders correctly 1`] = `
<div
  className="Component-container-2"
>
  <div
    className="Component-pembuka-3"
  />
  <div
    className="Component-add-4"
  />
</div>
`;
